### PR TITLE
update local-volume-provider to v0.3.1

### PR DESCRIPTION
**Pull request by laverya**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by emosbaugh** [comment]
Looks like we are using v0.3.0 in kurl https://github.com/replicatedhq/kURL/blob/04a733496c9c3e99793bc538ef4eb7d3cb9d25e2/addons/velero/1.7.1/Manifest#L6

**Comment by laverya** [comment]
v0.3.1 is the same code as v0.3.0, just a newer build

File: Makefile [hunk 1]
```diff
@@ -140,9 +140,6 @@ build-release:
 	mkdir -p bin/docker-archive/local-volume-provider
 	skopeo copy docker://replicated/local-volume-provider:${LVP_VERSION} docker-archive:bin/docker-archive/local-volume-provider/${LVP_VERSION}
 
-	mkdir -p bin/docker-archive/local-volume-fileserver
```
Comment (sgalsaleh) [inline Makefile:143]: why is this removed? i think this is needed for velero to function (getting backup logs, etc...)

File: pkg/print/fs-lvp-velero-config.go [hunk 2]
```diff
@@ -92,7 +92,7 @@ func printLvpFileSystemVeleroInstructions(c *LvpFileSystemVeleroConfig, log *log
 	  replicated.com/nfs: ObjectStore
 	  replicated.com/hostpath: ObjectStore
 	data:
-	  fileserverImage: %s/local-volume-fileserver:v0.1.0
```
Comment (sgalsaleh) [inline pkg/print/fs-lvp-velero-config.go:95]: so this is part of the `local-volume-provider` image now?

Comment (laverya) [reply pkg/print/fs-lvp-velero-config.go:95]: it is, yes

**Review by sgalsaleh — APPROVED** [review]

File: pkg/print/fs-lvp-velero-config.go (hunk 2 above)
Comment (laverya) [reply pkg/print/fs-lvp-velero-config.go:95]: Or at least the v0.2.0+ versions of `local-volume-provider` no longer include that image, and the velero plugin in kurl doesn't use it

**Review by sgalsaleh — APPROVED** [review]
